honeybadger-generate-sourcemaps.sh

```diff
@@ -13,11 +13,27 @@
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 PROJECT_ROOT_DIR="$SCRIPT_DIR/../../"
+INDEX_FILE="$PROJECT_ROOT_DIR/index.js"
+
+# Check if index.js file exists and if try to search for .ts or .tsx equivalent
+
+if [[ ! -f "$INDEX_FILE" ]]; then
+	if [[ -f "$PROJECT_ROOT_DIR/index.tsx" ]]; then
+		INDEX_FILE="$PROJECT_ROOT_DIR/index.tsx"
+	else
+		if [[ -f "index.ts" ]]; then
```

Comment:
Should this be `if [[ -f "$PROJECT_ROOT_DIR/index.ts" ]]; then` ?

Reply:
Yes, you're right!